test/suites/container_devices_nic_bridged_filtering.sh

```diff
@@ -35,7 +35,7 @@ test_container_devices_nic_bridged_filtering() {
 
   lxc network set "${brName}" ipv6.dhcp.stateful true
   lxc network set "${brName}" bridge.hwaddr 00:11:22:33:44:55
-  lxc network set "${brName}" ipv4.address 192.0.2.1/24
+  lxc network set "${brName}" ipv4.address 192.0.2.1/16
```

Comment:
Can you use the documentation subnets for the additional routes to avoid conflicts with real (public or private) networks.

https://en.wikipedia.org/wiki/Reserved_IP_addresses

To overcome the Network unreachable errors you were mentioning you can set them in the network's `ipv{n}.routes` setting and it will cause a route to be added into the bridge network.